x11-wm/selectwm: EAPI6, use HTTPS, update patch

mm1ke opened the pull request:
Hi,

This my take on updating this package towards EAPI6.

Please review.

Moiman commented on x11-wm/selectwm/selectwm-0.4.1.ebuild:
```diff
@@ -1,37 +1,35 @@
 		"${FILESDIR}"/${P}-glibc-2.10.patch \
 		"${FILESDIR}"/${P}-nostrip.patch
 	eautoreconf
+	default
```
move default before eautoreconf

mm1ke commented:
rebased, any chance we can merge this? :)

Moiman commented on x11-wm/selectwm/selectwm-0.4.1.ebuild:
```diff
@@ -1,27 +1,30 @@
 
 src_prepare() {
-	epatch "${FILESDIR}"/${P}-enable-deprecated-gtk.patch \
+	eapply "${FILESDIR}"/${P}-enable-deprecated-gtk.patch \
```
Can you use PATCHES=() ?

mm1ke replied:
Hi, i've updated the PR to use the PATCHES Variable

Moiman commented on x11-wm/selectwm/selectwm-0.4.1.ebuild:
```diff
@@ -1,27 +1,33 @@
+HOMEPAGE="https://ordiluc.net/selectwm"
+SRC_URI="https://ordiluc.net/selectwm/${P}.tar.bz2"
 
 LICENSE="GPL-2"
```
License is gpl-2+

mm1ke replied:
thanks for pointing that out, fixed it :)

flappyports commented:
@mm1ke Please make this a revbump as @mgorny asked and I will merge it.

mm1ke commented:
@flappyports @mgorny I've added a r1 with all the changes and only fixed the https for the original file. I've also dropped the stable keywords for the r1 and fixed all repoman warnings.
Please review.

mm1ke commented:
Sorry, i've accidentally removed this branch before ;)
